dts/riscv/gigadevice/gd32vf103.dtsi

```diff
@@ -25,6 +25,7 @@
 			clock-frequency = <DT_FREQ_M(108)>;
 			mcause-exception-mask = <0x7ff>;
 			compatible = "nuclei,bumblebee";
+			riscv,isa = "rv32imac_zicsr_zifencei";
```

Comment:
From [Bumblebee Core Architecture Manual](https://github.com/nucleisys/Bumblebee_Core_Doc/blob/master/Bumblebee%20Core%20Architecture%20Manual.pdf) section 1.2(page 12), gd32vf103 supported Instruction Subset is RV32IMAC.

Reply:
Hi @cameled. It looks like the document you linked here was written before Zicsr and Zifencei were separated and started being included in the ISA string, as it was published 4 years ago.

For example, gd32vf103 also selects these extensions in its [Kconfig.soc](https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/riscv/riscv-privileged/gd32vf103/Kconfig.soc#L16-L17) file.

Reply:
Ok, It's good for me now.